airflow_dags/wikipedia/ngrams/dolt_load.py

```diff
@@ -0,0 +1,354 @@
+import subprocess
```

Comment:
@katiemcculloch This is the file with all the load logic. Very high level what's happening is:
1. Download dump from wikipedia (XML format)
2. Use submodule `wikiextractor` which parses the XML into lines (strings)
3. Use the helper functions to extract unigrams (1 word), bigrams (2 words), and trigrams (3 words) and saves each to its own dictionary. The dictionary keeps track of how many times each ngram appears in total and how many articles it appears in
4. The dumps are too big to hold all those dictionaries, so for each ngram at the shard length specified, it writes each dictionary to a sorted csv and empties the dictionary
5. Once all the csvs have been written, it uses merge sort to combine all the small csvs into one big csv for each ngram
6. Using `doltpy` loaders, each of these csvs is loaded into Dolt and pushed to Dolthub.

There have been newer PRs since this one, but basically where it failed is when it's importing to Dolt (you can see the log [here](https://airflow.awsdev.ld-corp.com/admin/airflow/log?dag_id=wikipedia-ngrams-backfill&task_id=backfill-data-20190901&execution_date=2019-12-18T02%3A55%3A52.293306%2B00%3A00)

It might be a problem with `doltpy`, but the error messages have not been useful so we're not sure